linetools/isgm/io.py

```diff
@@ -41,3 +51,83 @@ def abssys_from_json(filename):
 
     # Return
     return abs_sys
+
+
+def read_joebvp(filename, coord, llist=None, specfile=None, chk_vel=False):
+    """
```

Comment:
Wasn't a version of this already coded in package `joebvp`?

Reply:
I don't know.  @jnburchett ?